src/redux/wallets.js

```diff
@@ -224,6 +227,56 @@ export const createAccountForWallet = (id, color, name) => async (
   return newWallets;
 };
 
+export const getWalletENSAvatars = async (walletsState, dispatch) => {
+  const { wallets, walletNames, selected } = walletsState;
```

Comment:
is `selected` supposed to be `selectedWallet` ? (e.g, take a look at `useWalletENSAvatar` usage of the `getWalletENSAvatars` function)

Reply:
from what I see they're the same, only naming is different inside the hook https://github.com/rainbow-me/rainbow/blob/develop/src/hooks/useWallets.js#L19 

for that specific method, we can't use the hook so we're just passing the `wallets` state which contains the `selected` param which is the same one from the hook, just different name

Reply:
for reference: now fixed in: https://github.com/rainbow-me/rainbow/pull/3392/commits/9f23d3a336997d8571e08a001e27264559f8ce40